codemeta.json

```diff
@@ -0,0 +1,178 @@
+{
+  "@context": ["https://doi.org/doi:10.5063/schema/codemeta-2.0", "http://schema.org"],
+  "@type": "SoftwareSourceCode",
+  "identifier": "rbison",
+  "description": "Interface to the 'USGS' 'BISON' (<https://bison.usgs.gov/>)\n    'API', a 'database' for species occurrence data. Data comes from\n    species in the United States from participating data providers. You can get\n    data via 'taxonomic' and location based queries. A simple function\n    is provided to help visualize data.",
+  "name": "rbison: Interface to the 'USGS' 'BISON' 'API'",
+  "codeRepository": "https://github.com/chrisfan24/rbison.git",
```

Comment:
looks like codemetar is detecting the git url for your fork; rather than the original package, as the codeRepository, so we'll need to fix that.